packages/graphql-server/src/resolvers/appointment/types.graphql

```diff
@@ -0,0 +1,238 @@
+# Types
```

Comment:
Pls remove redundant things and need to check any required params in backend and input `!` for that params

Reply:
Can we auto generated these types from TS defintions?